src/test/java/tests/DownloadAppTest.java

```diff
@@ -0,0 +1,64 @@
+package tests;
+
+import baseEntities.BaseTest;
+import core.PropertyReader;
+import io.qameta.allure.Description;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.AboutPage;
+import steps.MainPageSteps;
+import testData.StaticProvider;
+
+import java.io.File;
+import java.util.Objects;
+
+public class DownloadAppTest extends BaseTest {
+
+    @Test(dataProvider = "InstallAppNames", dataProviderClass = StaticProvider.class)
+    @Description("Download steam installation app for any available platform.")
+    public void downloadWinAppTest(String linkFileName, String downloadedFileName) throws InterruptedException {
```

Comment:
90% of the code that this test contains should be moved to different methods. 
For work with files should be created a new class